app/tests/unit/components/Dashboard.test.tsx

```diff
@@ -40,6 +41,8 @@ const componentTestingHelper = new ComponentTestingHelper<DashboardTestQuery>({
   defaultComponentProps: {},
 });
 
+const supportEmail = getConfig()?.publicRuntimeConfig?.SUPPORT_EMAIL;
```

Comment:
we should mock this value instead of using the real one here
```suggestion
jest.mock("next/config");
mocked(getConfig).mockImplementation(() => ({
  publicRuntimeConfig: {
    SUPPORT_EMAIL: "test@email.com"
  }
}));
```

and add the test email in the jest expectation